pkgs/development/python-modules/python-telegram-bot/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage, isPy3k, certifi, future }:
+
+buildPythonPackage rec {
+  pname = "python-telegram-bot";
+  version = "9.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0a5b4wfc6ms7kblynw2h3ygpww98kyz5n8iibqbdyykwx8xj7hzm";
+  };
+
+  propagatedBuildInputs = [ certifi future ];
+  doCheck = !isPy3k;
```

Comment:
What is the reason for this? Can you include a comment.

Reply:
The reason is that 0 tests are run when using Python 2, so it should really be just `doCheck = false` for both.
But I made one test run by adding `pysocks` as a dependency. I believe this is possible for the 7 remaining failing tests as well.

Reply:
Turns out the tests need network access (obviously). So it should be `doCheck = false`.